src/webui/www/private/views/preferences.html

```diff
@@ -942,7 +942,7 @@
                     <label for="saveResumeDataInterval">QBT_TR(Save resume data interval:)QBT_TR[CONTEXT=OptionsDialog]</label>
                 </td>
                 <td>
-                    <input type="text" id="saveResumeDataInterval" style="width: 15em;">&nbsp;&nbsp;QBT_TR(min)QBT_TR[CONTEXT=OptionsDialog]
+                    <input type="text" id="saveResumeDataInterval" style="width: 15em;" />&nbsp;&nbsp;QBT_TR(min)QBT_TR[CONTEXT=OptionsDialog]
```

Comment:
I think this change goes in different way of current coding style, or are there any rationale behind it?
If not it is better to revert them.

Or they should be applied globally and not just here.

Reply:
I noticed other ones followed this style in preference.html and hence tried to keep it uniform.

Reply:
>I noticed other ones followed this style in preference.html and hence tried to keep it uniform.

OK. Later I can try to enforce the coding style with automatic checks. But for now, I would suggest reverting them.